runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -42,11 +44,15 @@
     private final String restoreTempDirPath;
 
     // The stream IDs of tables which are restored
-    private List<UUID> streamIDs;
+    private List<String> tableBackups;
 
     // The Corfu Store associated with the runtime
     private CorfuStore corfuStore;
 
+    // If set true, will clean up all tables before restore;
+    // If set false, will only clean up tables that are to be restored
+    private boolean clearAllTablesBeforeRestore = false;
+
```

Comment:
nit: is this initialization needed here? also, I thought `tableBackups`, `corfuStore` and `clearAllTablesBeforeRestore` could be set to final

Reply:
You are right. Removed the initial value.